src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/requirements.txt

```
@@ -1,54 +1,6 @@
 # Install libs here that you might want in AWS CodeBuild
-aws-cdk-assets~=1.188.0
-aws-cdk.aws-apigateway~=1.188.0
-aws-cdk.aws-applicationautoscaling~=1.188.0
-aws-cdk.aws-autoscaling_common~=1.188.0
-aws-cdk.aws-autoscaling_hooktargets~=1.188.0
-aws-cdk.aws-autoscaling~=1.188.0
-aws-cdk.aws-certificatemanager~=1.188.0
-aws-cdk.aws-chatbot~=1.188.0
-aws-cdk.aws-cloudformation~=1.188.0
-aws-cdk.aws-cloudfront~=1.188.0
-aws-cdk.aws-cloudwatch~=1.188.0
-aws-cdk.aws-codebuild~=1.188.0
-aws-cdk.aws-codecommit~=1.188.0
-aws-cdk.aws-codedeploy~=1.188.0
-aws-cdk.aws-codepipeline-actions~=1.188.0
-aws-cdk.aws-codepipeline~=1.188.0
-aws-cdk.aws-dynamodb~=1.188.0
-aws-cdk.aws-ec2~=1.188.0
-aws-cdk.aws-ecr_assets~=1.188.0
-aws-cdk.aws-ecr~=1.188.0
-aws-cdk.aws-ecr~=1.188.0
-aws-cdk.aws-ecs~=1.188.0
-aws-cdk.aws-elasticloadbalancingv2~=1.188.0
-aws-cdk.aws-elasticloadbalancing~=1.188.0
-aws-cdk.aws-events_targets~=1.188.0
-aws-cdk.aws-events~=1.188.0
-aws-cdk.aws-iam~=1.188.0
-aws-cdk.aws-kinesis~=1.188.0
-aws-cdk.aws-kms~=1.188.0
-aws-cdk.aws-lambda_event_sources~=1.188.0
-aws-cdk.aws-lambda~=1.188.0
-aws-cdk.aws-logs~=1.188.0
-aws-cdk.aws-route53_targets~=1.188.0
-aws-cdk.aws-route53~=1.188.0
-aws-cdk.aws-s3_assets~=1.188.0
-aws-cdk.aws-s3_assets~=1.188.0
-aws-cdk.aws-s3_notifications~=1.188.0
-aws-cdk.aws-s3~=1.188.0
-aws-cdk.aws-sam~=1.188.0
-aws-cdk.aws-secretsmanager~=1.188.0
-aws-cdk.aws-servicediscovery~=1.188.0
-aws-cdk.aws-sns_subscriptions~=1.188.0
-aws-cdk.aws-sns~=1.188.0
-aws-cdk.aws-sns~=1.188.0
-aws-cdk.aws-sqs~=1.188.0
-aws-cdk.aws-ssm~=1.188.0
-aws-cdk.aws-stepfunctions~=1.188.0
-aws-cdk.core~=1.188.0
-aws-cdk.cx-api~=1.188.0
-aws-cdk.region-info~=1.188.0
+aws-cdk-lib==2.68.0
```

Apologies for the delayed second review, in the meanwhile a few new CDK releases were published, could you update this to:

```suggestion
aws-cdk-lib==2.75.1
```